zero/gkyl_range.h

```diff
@@ -428,6 +434,19 @@ bool gkyl_range_is_on_lower_edge(int dir, const struct gkyl_range *range,
 bool gkyl_range_is_on_upper_edge(int dir, const struct gkyl_range *range,
   const struct gkyl_range *parent);
 
+/**
+ * Check if @a targ range shares an edge with the @a base range. The
```

Comment:
Is there a reason this comment does @a for the definitive article before targ and base?

Reply:
@a is the syntax for doxygen to say the thing that follows it is a parameter